src/components/add-new-todo.vue

```diff
@@ -18,7 +18,7 @@ export default Vue.component('AddNewTodo', {
     clearTodoInput () {
```

Comment:
change naming to correct one

Reply:
clearTodoInput()  changed to createNewTodo().